### PR TITLE
Destroys the Mind Reader mutation

### PR DESCRIPTION
**Pull request by athena413**
# General Documentation

### Intent of your Pull Request

See title - Mind Reader mutation removed from the game

### Why is this change good for the game?

The power exists ONLY to fuck over antags - with only a 20% chance of you even knowing you've been screwed. I see NO justification for this power's existence.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Mind Reader mutation is gone.

### What should players be aware of when it comes to the changes your PR is implementing?

Antags will be fucked over less by genetics powergame

### What general grouping does this PR fall under?
Genetics - removal

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No

# Changelog

:cl:  
rscdel: Mind Reader mutation obliterated.
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
genetics gonna end up with only 2 positive mutations soon lmao

**Comment by Ratvar** [comment]
this is what happens when nobody thinks what they add

**Comment by UselessTheremin** [comment]
also it wont compile, remove "mindread" from
code/game/objects/items/dna_injector.dm 
and
code/datums/mutations/_combined.dm

**Review by redmoogle — CHANGES_REQUESTED** [review]
_maps/map_files/generic/CentCom.dmm:20890:error: undefined type: /obj/item/dnainjector/mindread

You forgot to get rid of it in the centcomm test chamber

**Comment by SomeguyManperson** [comment]
remove it from the map so it compiles
